dist/tools/esptools/export.sh

```diff
@@ -15,6 +22,47 @@ fi
 
 TOOLS_PATH="${IDF_TOOLS_PATH}/tools"
 
+# this function expects the parameters $TOOL, $TOOLS_DIR and $*_VERSION
+export_checks()
+{
+    TOOL="$1"
+    TOOLS_DIR_INT="$2" # internal TOOLS_DIR
+    TOOLS_VERSION="$3"
+
+    # create the wildcard expression from the TOOLS_DIR
+    TOOLS_DIR_BASE=$(echo "$TOOLS_DIR_INT/bin" | sed "s|/$TOOLS_VERSION/|/[^/]*/|")
+    TOOLS_DIR_IN_PATH=$(echo "$PATH" | grep "${TOOLS_DIR_INT}")
+
+    if [ ! -e "${TOOLS_DIR_INT}" ]; then
+        echo "${TOOLS_DIR_INT} does not exist - please run"
+        echo "\${RIOTBASE}/dist/tools/esptools/install.sh $TOOL"
+        return
```

Comment:
We need the return value on error for OpenOCD settings, see below:
```suggestion
        return 1
```